_layouts/index.html

```diff
@@ -116,6 +116,17 @@ <h3>And much more</h3>
 		<h2 class="text-center top-space">Recent articles about Skydive</h2>
 		<br>
 
+		<div class="row">
+			<div class="col-sm-12">
+				<h3><a href="https://www.youtube.com/watch?v=jl2_jUKspJA">
+					k8s configuration roll-out using Skydive
+				</a></h3>
+				<p>
+In this demo we show how skydive (http://skydive.network) can help roll-out configuration changes across your k8s landscape, helping in visualizing which pods are dependent on the configuration object and going beyond that by automatic the restarting of all dependent pods automatically by utilizing Skydive REST API.
```

Comment:
s/skydive/Skydive/

Reply:
I don't think we need to put a link to Skydive as we already are on the Skydive website.

Reply:
ok. ok.